src/main.rs

```diff
@@ -126,6 +126,23 @@ fn main() -> Result<()> {
         exit(0);
     }
 
+    if matches.is_present(Other_commands::whoami) {
+        if let Some(_value) = matches.subcommand_matches("token") {
+            openid::print_token(&context);
+        } else {
+            println!("cluster adress : {}", context.drogue_cloud_url);
+            println!(
+                "Default App : {}",
+                match context.default_app {
+                    None => "No default app",
+                    Some(ref x) => x,
+                }
+            );
+            util::print_version(&Ok(config));
```

Comment:
Please move this logic to the `util` module. main is already quite long and messy so it's better if we keep only the arg-related suff here IMO.